src/cairoUtilFuncGen/storage/storageToMemory.ts

```diff
@@ -265,7 +293,7 @@ export class StorageToMemoryGen extends StringIndexedFuncGen {
 
         `func ${funcName}${implicits}(loc : felt) -> (mem_loc : felt):`,
         `    alloc_locals`,
-        `    let (name) = WARP_STORAGE.read(loc)`,
+        `    let (name) = readId(loc)`,
```

Comment:
Where is this function defined/imported? I searched the file and couldn't find it.